sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeHealthcareEntitiesTests.cs

```diff
@@ -62,26 +63,23 @@ public async Task RecognizeHealthcareEntitiesTest()
                 {
                     var linksList = new List<string> { "UMLS", "AOD", "ATC", "CCPSS", "CHV", "CSP", "DRUGBANK", "GS", "LCH_NW", "LNC", "MEDCIN", "MMSL", "MSH", "MTHSPL", "NCI", "NCI_CTRP", "NCI_DCP", "NCI_DTP", "NCI_FDA", "NCI_NCI-GLOSS", "NDDF", "PDQ", "RCD", "RXNORM", "SNM", "SNMI", "SNOMEDCT_US", "USP", "USPMG", "VANDF" };
 
-                    foreach (HealthcareEntityLink link in entity.Links)
-                        Assert.IsTrue(linksList.Contains(link.DataSource));
+                    foreach (EntityDataSource entityDataSource in entity.DataSources)
+                        Assert.IsTrue(linksList.Contains(entityDataSource.Name));
                 }
-            }
 
-            foreach (HealthcareRelation relation in result.Relations)
-            {
-                if (relation.RelationType == "DosageOfMedication")
+                if (entity.Text == "100mg")
                 {
-                    Assert.AreEqual(relation.Source.Text, "100mg");
-                    Assert.AreEqual(relation.Source.Category, "Dosage");
-                    Assert.AreEqual(relation.Source.ConfidenceScore, 1);
-                    Assert.AreEqual(relation.Source.Length, 5);
-                    Assert.AreEqual(relation.Source.Offset, 18);
-
-                    Assert.AreEqual(relation.Target.Text, "ibuprofen");
-                    Assert.AreEqual(relation.Target.Category, "MedicationName");
-                    Assert.AreEqual(relation.Target.ConfidenceScore, 1);
-                    Assert.AreEqual(relation.Target.Length, 9);
-                    Assert.AreEqual(relation.Target.Offset, 27);
+                    HealthcareEntity relatedEntity= new HealthcareEntity(new HealthcareEntityInternal("ibuprofen", "MedicationName", 27, 0, 1.0, false));
+
+                    Assert.IsTrue(entity.RelatedEntities.Count == 1);
+                    Assert.AreEqual(relatedEntity.Text, entity.RelatedEntities.ElementAt(0).Key.Text);
```

Comment:
you could always do
```
Assert.IsTrue(entity.RelatedEntities.Count == 1);
var relatedEntity = entity.RelatedEntities.FirstOrDefault().Key;`
Assert.AreEqual("ibuprofen", relatedEntity.Text);
```

it will make it easier to read